flocker_bb/builders/flocker_vagrant.py

```diff
@@ -573,6 +596,24 @@ def getSchedulers():
                         or MergeForward._isRelease(branch)),
             )
         ),
+        timed.Nightly(
+            name="flocker-build-vagrant-basebox-dev-box",
+            builderNames=['flocker-build-vagrant-basebox-dev-box'],
+            codebases={
+                "flocker": {"repository": GITHUB + b"/flocker"},
+            },
+            branch=ChangeFilter(
+                branch_fn=lambda branch:
+                    (MergeForward._isMaster(branch)
+                        or MergeForward._isRelease(branch)),
+            ),
+            dayOfWeek=range(0, 7),
+            hour=5,
+            minute=7,
+            onlyIfChanged=False
```

Comment:
Probably there is no need to build a new box, if nothing has changed. And perhaps we only care if some files have changed, but we can address that in a follow up ticket.